backend/functions/src/index.ts

```diff
@@ -137,4 +137,9 @@ export const deleteTask = functions.https
     }
     );
 
+export const addTask = functions.https
+    .onRequest(async (request, response) => {
+      
```

Comment:
To get a feel for the request/response shape, I recommend using the command: ``firebase emulators:start``, and then console.log(request, response), to see what each thing looks like. They will be filled with data that is irrelevant, but it is good to get a feel for how req/res HTTP stuff works. 

Note that you'll need to run ``npm run build`` in the functions/ directory,
then go back into backend/ with ``cd ..``
then you can run ``firebase emulators:start``, a good way to test your implementation.